src/components/charts/ByMonthColumnChart.js

```diff
@@ -0,0 +1,54 @@
+import React, { Fragment, useMemo, useState } from "react";
+import PropTypes from "prop-types";
+import { ColumnChart, Model } from "@gooddata/react-components";
+import moment from "moment";
+
+// COMPONENTS
+import MonthSelect from "../selects/MonthSelect";
+
+// CONSTANTS
+import { CHART_PARAMS, DATE_FORMAT } from "../../constants/chartParams";
+
+const ByMonthColumnChart = ({ year }) => {
+  const [selectedMonth, setSelectedMonth] = useState(1);
+
+  const measures = useMemo(() => [
```

Comment:
`measures` should be passed in as a prop so that the component can be re-used to render charts for different measures.